org.swordpuffin.splices.metainfo.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+	<id>org.swordpuffin.splices.desktop</id>
+  <name>Splices</name>
+	<metadata_license>CC0-1.0</metadata_license>
+	<project_license>GPL-3.0-or-later</project_license>
+  <summary>A small time killer where you make words out of randomly generated letters. You can play in either free play mode, free play with a timer, or normal mode where the goal is maximize the length of the words</summary>  <screenshots>
+    <screenshot type="default">
+      <image>https://github.com/SwordPuffin/Splices/blob/master/data/Screenshot.png</image>
```

Comment:
these and below:
1. it's not a link to the image, but to the github page
2. it should never be a link to the branch but to a specific revision